data/changelog.txt

```diff
@@ -222,6 +222,9 @@ Adds a way to fill shallow water tiles.
 Changes mechanics of MUSHY flag from from causing insta-rot on next thaw to making rot progress faster.
 Abandon camp NPC talk and bulletin board mission.
 Scale learning speed based on INT stat.
+Added ups_charges_modifier handling for gunmods.
+Adds random town name string.
+Allow professions to start with a nearby vehicle - trucker profession.
```

Comment:
Script needs to be aware of merge target branch - some lines here are from `dev` branch.

Reply:
If it would help, here is link with list of PRs merged to `dev` branch - https://github.com/CleverRaven/Cataclysm-DDA/issues/36949#issuecomment-595704738